package.json

```diff
@@ -61,6 +61,7 @@
   },
   "dependencies": {
     "google-gax": "^0.18.0",
+    "lodash": "^4.17.10",
```

Comment:
Is this being used?

Reply:
From the language_service_client.js it's used just once i believe (just did a quick find); but that wasn't part of this new automl branch

Reply:
Did you find it in the master branch, or somewhere else? I found it there on master, but it's the `lodash.merge` dependency (https://github.com/googleapis/nodejs-language/blob/bc15088e119e22f4368afe5cdd3245e062d72335/src/v1/language_service_client.js#L19).